apps/app/src/app/pages/(components)/components/(tabs)/tabs--vertical.preview.ts

```diff
@@ -49,12 +49,12 @@ import { HlmTabsContentDirective, HlmTabsListDirective, HlmTabsTriggerDirective
 	},
 	template: `
 		<brn-tabs class="mx-auto flex max-w-3xl flex-row space-x-2" orientation="vertical" value="account">
-			<brn-tabs-list hlmTabsList orientation="vertical" aria-label="tabs example">
-				<button class="w-full" hlmTabsTrigger brnTabsTrigger="account">Account</button>
-				<button class="w-full" hlmTabsTrigger brnTabsTrigger="password">Password</button>
-				<button class="w-full" hlmTabsTrigger brnTabsTrigger="danger">Danger Zone</button>
+			<brn-tabs-list hlm orientation="vertical" aria-label="tabs example">
```

Comment:
Is it possible to turn this into hlm-tabs-list and not even need the directive AND component?
Sometimes that is easy to achieve, sometimes it is a pain in the a**, but if it's possible without too much headaches I think it would be nice

Reply:
I tried that too. 
It would be possible if I copy the template from brn, but I didn't like that. 
AFAIK it is not possible to reuse the Brn Template and there is no such thing as HostComponent. ;) 
https://github.com/angular/angular/issues/47900

Reply:
Couple crazy ideas I had today. Declare and export the template as a string variable inside the brn-tab-list component and reuse in a hlm component, then extend the brn-tabs-list component. Not sure how big of a pain that is though. Overall, I think this is good enough. I think we are simply limited by Angular here

Reply:
The other one is to move as much of the functionality as possible into a separate directive, which can be used with hostdirectives and then wire up the rest manually

Reply:
Sorry for the confusion, it was only a ngcontent component. 
I must have confused it with something else i was looking to improve. 
That means, we can easily make a directive and use a hostdirective. 
But maybe the tips help me for my other problem. ;) 
i will change it an push it soon.